scripts/qesap/lib/cmds.py

```diff
@@ -173,26 +173,29 @@ def cmd_terraform(configure_data, base_project, dryrun, workspace='default', des
     if not cfg_paths:
         return Status(f"Invalid folder structure at {base_project}")
 
+    terraform_common_cmd = f"{config.get_terraform_bin()} -chdir={cfg_paths['provider']}"
+
     cmds = []
-    terraform_common_cmd = [config.get_terraform_bin(), f"-chdir={cfg_paths['provider']}"]
     if destroy:
-        cmds.append(terraform_common_cmd + ['destroy', '-auto-approve', '-no-color'])
+        cmds.append(f"{terraform_common_cmd} destroy -auto-approve")
         if workspace != 'default':
-            cmds.append(terraform_common_cmd + ['workspace', 'select', 'default', '-no-color'])
-            cmds.append(terraform_common_cmd + ['workspace', 'delete', workspace, '-no-color'])
+            cmds.append(f"{terraform_common_cmd} workspace select default")
+            cmds.append(f"{terraform_common_cmd} workspace delete {workspace}")
     else:
-        cmds.append(terraform_common_cmd + ['init', '-no-color'])
+        cmds.append(f"{terraform_common_cmd} init")
         if workspace != 'default':
-            cmds.append(terraform_common_cmd + ['workspace', 'new', workspace, '-no-color'])
-        cmds.append(terraform_common_cmd + ['plan', '-out=plan.zip', '-no-color'])
-        cmds.append(terraform_common_cmd + ['apply', '-auto-approve', 'plan.zip', '-no-color'])
+            cmds.append(f"{terraform_common_cmd} workspace new {workspace}")
+        cmds.append(f"{terraform_common_cmd} plan -out=plan.zip")
+        cmds.append(f"{terraform_common_cmd} apply -auto-approve plan.zip")
+
     for command in cmds:
+        command += ' -no-color'
         if dryrun:
-            print(' '.join(command))
+            print(command)
         else:
             ret, out = lib.process_manager.subprocess_run(command)
             log.debug("Terraform process return ret:%d", ret)
-            log_filename = f"terraform.{command[2]}.log.txt"
+            log_filename = f"terraform.{command.split()[2]}.log.txt"
```

Comment:
How was this working before? Was `command` a list ... doesn't look like it from L191.

Reply:
It was `command = ['terraform', 'init']` and this PR change it to `command = 'terraform init'`